directions/models.py

```diff
@@ -679,6 +679,8 @@ def gen_napravleniya_by_issledovaniya(
         childrens = {}
         researches_grouped_by_lab = []  # Лист с выбранными исследованиями по лабораториям
         lab_podrazdeleniye_pk = list(Podrazdeleniya.objects.values_list('pk', flat=True).filter(p_type=2))
+        if not current_global_direction_params.get("title", False):
```

Comment:
```suggestion
        if current_global_direction_params  and not current_global_direction_params.get("title", False):
```

Иначе всё упадёт. Код не безопасный.